src/librustc/mir/mod.rs

```diff
@@ -471,6 +464,7 @@ pub enum TerminatorKind<'tcx> {
 
         /// Possible values. The locations to branch to in each case
         /// are found in the corresponding indices from the `targets` vector.
+        // FIXME: ConstVal doesn’t quite make any sense here? Its a Switch*Int*.
         values: Vec<ConstVal>,
 
         /// Possible branch sites. The length of this vector should be
```

Comment:
Current setup is nice as the last block always gets considered an `otherwise` (without much effort, as you describe), but is also easy to overlook.